diesel_codegen/src/embed_migrations.rs

```diff
@@ -16,7 +16,7 @@ pub fn derive_embed_migrations(input: syn::MacroInput) -> quote::Tokens {
 
     let options = get_options_from_input(&input.attrs, bug);
     let migrations_path_opt = options.map(|o| get_option(&o, "migrations_path", bug));
-    let migrations_expr = migration_directory_from_given_path(migrations_path_opt)
+    let migrations_expr = migration_directory_from_given_path(migrations_path_opt.as_ref().map(|s| s.as_str()))
```

Comment:
Not that I know of. I'd probably replace `.map(|s| s.as_str())` with `.map(String::as_ref)` though.

Edit: There even is a [SO question](http://stackoverflow.com/q/31233938/1254484) about that with basically the same answer.